2021-11-26 00:08:21,578 INFO    MainThread:33629 [wandb_setup.py:_flush():71] setting env: {}
2021-11-26 00:08:21,579 INFO    MainThread:33629 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-26 00:08:21,579 INFO    MainThread:33629 [wandb_init.py:_log_setup():371] Logging user logs to /opt/ml/code/wandb/run-20211126_000821-2gw5nq2t/logs/debug.log
2021-11-26 00:08:21,580 INFO    MainThread:33629 [wandb_init.py:_log_setup():372] Logging internal logs to /opt/ml/code/wandb/run-20211126_000821-2gw5nq2t/logs/debug-internal.log
2021-11-26 00:08:21,580 INFO    MainThread:33629 [wandb_init.py:init():404] calling init triggers
2021-11-26 00:08:21,580 INFO    MainThread:33629 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2021-11-26 00:08:21,580 INFO    MainThread:33629 [wandb_init.py:init():449] starting backend
2021-11-26 00:08:21,580 INFO    MainThread:33629 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-11-26 00:08:21,591 INFO    MainThread:33629 [backend.py:ensure_launched():199] starting backend process...
2021-11-26 00:08:21,598 INFO    MainThread:33629 [backend.py:ensure_launched():205] started backend process with pid: 33665
2021-11-26 00:08:21,599 INFO    MainThread:33629 [wandb_init.py:init():458] backend started and connected
2021-11-26 00:08:21,600 INFO    MainThread:33629 [wandb_init.py:init():519] updated telemetry
2021-11-26 00:08:21,601 INFO    MainThread:33629 [wandb_init.py:init():549] communicating current version
2021-11-26 00:08:22,439 INFO    MainThread:33629 [wandb_init.py:init():554] got version response 
2021-11-26 00:08:22,439 INFO    MainThread:33629 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-11-26 00:08:22,677 INFO    MainThread:33629 [wandb_init.py:init():592] starting run threads in backend
2021-11-26 00:08:27,623 INFO    MainThread:33629 [wandb_run.py:_console_start():1816] atexit reg
2021-11-26 00:08:27,624 INFO    MainThread:33629 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2021-11-26 00:08:27,624 INFO    MainThread:33629 [wandb_run.py:_redirect():1695] Redirecting console.
2021-11-26 00:08:27,626 INFO    MainThread:33629 [wandb_run.py:_redirect():1751] Redirects installed.
2021-11-26 00:08:27,626 INFO    MainThread:33629 [wandb_init.py:init():619] run started, returning control to user process
2021-11-26 01:15:18,193 INFO    MainThread:33629 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-11-26 01:15:18,193 INFO    MainThread:33629 [wandb_run.py:_restore():1758] restore
2021-11-26 01:15:20,514 INFO    MainThread:33629 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 507
  total_bytes: 507
}

2021-11-26 01:15:20,760 INFO    MainThread:33629 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 507
  total_bytes: 507
}

2021-11-26 01:15:21,510 INFO    MainThread:33629 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  uploaded_bytes: 507
  total_bytes: 93754
}

2021-11-26 01:15:21,612 INFO    MainThread:33629 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 507
  total_bytes: 97664
}

2021-11-26 01:15:21,714 INFO    MainThread:33629 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 507
  total_bytes: 97664
}

2021-11-26 01:15:21,816 INFO    MainThread:33629 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 97664
  total_bytes: 97664
}

2021-11-26 01:15:21,919 INFO    MainThread:33629 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 97664
  total_bytes: 97664
}

2021-11-26 01:15:22,021 INFO    MainThread:33629 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 97664
  total_bytes: 97664
}

2021-11-26 01:15:22,123 INFO    MainThread:33629 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 97664
  total_bytes: 97664
}

2021-11-26 01:15:22,225 INFO    MainThread:33629 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 97664
  total_bytes: 97664
}

2021-11-26 01:15:22,729 INFO    MainThread:33629 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 97664
  total_bytes: 97664
}

2021-11-26 01:15:23,199 INFO    MainThread:33629 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 97664
  total_bytes: 97664
}
local_info {
}

2021-11-26 01:15:24,385 INFO    MainThread:33629 [wandb_run.py:_append_files():2184] logging synced files
